content/en/pod.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## What it is
+
+In [cloud native computing](/cloud-computing/), a "pod" is a basic building block for deploying and managing containerized applications. 
+A pod represents a single instance of an application, and can contain one or more [containers](/container/).
+[Kubernetes](/kubernetes/) manages pods as part of a larger deployment, and can scale pods up or down depending on the application's resource requirements.
+
+## Problem it addresses
+
+Containers provide a lightweight and efficient way to package and deploy applications, but they can't do everything on their own. 
+Applications often require additional resources and services, such as storage or networking, which need to be managed and configured separately.
+
+This is where pods come in. A pod provides a way to group containers together and manage them as a single unit. 
+This makes it easier to ensure that all the resources and services in application needs are available and configured correctly.
+
+## How it helps
+
+Isolation: Each pod has its own isolated environment, including its own network stack and file system. This ensures that containers in the same pod can't interfere with each other, and helps maintain security and stability.
+
+Resource sharing: Containers in the same pod can share resources, such as CPU and memory, which allows for efficient use of resources.
+
+Simplified management: Pods make it easier to manage groups of containers as a single unit, rather than managing each container individually. 
+This can simplify deployment, scaling, and monitoring of applications.
+
+Portability: Pods can be easily moved between different hosts or cloud providers, which allows for greater flexibility and portability of applications.
```

Comment:
```suggestion
Since pods are isolated environments, they are [portable](/portability/). Meaning, they can be easily moved between hosts and/or cloud providers, providing operators with flexibility. 
```

Reply:
tbh I don't know if we want to say this. We are comparing pods to containers to show the advantage of pods over containers. But is the portability aspect not also true for containers?! ... Also I don't think that pods can be easily moved between cloud providers and hosts, they can be moved between k8s nodes.

Reply:
What should we say? I removed host and cloud provider.

Reply:
I agree - I'm not sure pods are exactly easily movable (you can move them, but there's downtime involved). Would it be better to discuss workloads in this context? (or would that introduce another concept that we may not want to bring up here...)

Reply:
Ok, so we just remove the sentence?